tests/test_sca.py

```diff
@@ -138,15 +138,15 @@ def test_download_lookaside_sources(
     )
     print(f"mocking call to {expected_url}")
     requests_mock.get(expected_url, text="resp")
+
     downloaded_sources = _download_lookaside_sources(
-        distgit_source_archive, package_name, package_type
+        distgit_source_archive, 1, package_type, package_name
     )
     if expected_filename:
-        full_expected_filename = (
-            f"tests/data/lookaside/{package_type}/{package_name}/{expected_filename}/"
-        )
+        expected_hash_prefix = expected_path.split("/")[1][:6]
+        full_expected_filename = f"/tmp/lookaside/1/{expected_hash_prefix}-{expected_filename}/"
```

Comment:
Let's import the lookaside dir constant from sca.py (or make it a Django setting and import that) instead of hardcoding here.